cmd/commands/runtime.go

```diff
@@ -781,6 +781,28 @@ func createGitIntegration(ctx context.Context, opts *RuntimeInstallOptions) erro
 	return nil
 }
 
+func removeGitIntegrations(ctx context.Context, opts *RuntimeUninstallOptions) error {
+	appProxyClient, err := cfConfig.NewClient().AppProxy(ctx, opts.RuntimeName, store.Get().InsecureIngressHost) // why insecure ingress host
+	if err != nil {
+		return fmt.Errorf("failed to build app-proxy client: %w", err)
+	}
+
+	integrations, err := appProxyClient.GitIntegrations().List(ctx)
+	if err != nil {
+		return fmt.Errorf("failed to get list of git integrations: %w", err)
+	}
+
+	for _, intg := range integrations {
+		if err = RunGitIntegrationRemoveCommand(ctx, appProxyClient, intg.Name); err != nil {
+			command := util.Doc(fmt.Sprintf("\t<BIN> integration git remove %s", intg.Name))
+	
+			return fmt.Errorf(`%w. You can try to create it manually by running: %s`, err, command)
```

Comment:
The message here is wrong. and actually I think you can just straight up return the error from `RunGitIntegrationRemoveCommand` because it's probably descriptive enough...